lib/src/rules/omit_local_variable_types.dart

```diff
@@ -104,9 +103,8 @@ class _Visitor extends SimpleAstVisitor<void> {
       var iterableType = loopParts.iterable.staticType;
       if (iterableType is InterfaceType) {
         // TODO(srawlins): Is `DartType.asInstanceOf` the more correct API here?
-        var iterableInterfaces = iterableType.implementedInterfaces.where(
-            (type) =>
-                DartTypeUtilities.isInterface(type, 'Iterable', 'dart.core'));
+        var iterableInterfaces = iterableType.implementedInterfaces
+            .where((type) => type.isSameAs('Iterable', 'dart.core'));
```

Comment:
Not required, of course, but this really ought to be using `type.isDartCoreIterable`.

Reply:
Good catch! Fixed